packages/core/src/renderer/components/+namespaces/remove-subnamespace.injectable.ts

```diff
@@ -0,0 +1,46 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import customResourceDefinitionStoreInjectable from "../+custom-resources/definition.store.injectable";
+import apiManagerInjectable from "../../../common/k8s-api/api-manager/manager.injectable";
+import loggerInjectable from "../../../common/logger.injectable";
+
+const removeSubnamespaceInjectable = getInjectable({
+  id: "remove-subnamespace",
+
+  instantiate: (di) => {
+    const crdStore = di.inject(customResourceDefinitionStoreInjectable);
+    const apiManager = di.inject(apiManagerInjectable);
+    const logger = di.inject(loggerInjectable);
+
+    /**
+     * Removing subnamespace controlled by hierarchical namespace controller by deleting
+     * the SubnamespaceAnchor from the parent namespace. Anchor is CRD with the same name
+     * https://github.com/kubernetes-sigs/hierarchical-namespaces/blob/master/docs/user-guide/quickstart.md#subnamespaces-deep-dive
+     * @param subnamespaceName namespace name
+     * @returns 
+     */
+    return async (subnamespaceName: string) => {
+      await crdStore.loadAll();
```

Comment:
This seems unnecessary to do every single time especially since we are already always subscribed to CRDs for the sidebar

Reply:
We can't find proper CRD instance without this in store.

Reply:
When in tests or not?

Reply:
Not only in tests.

Reply:
I really don't think we should be loading all in a delete, especially not twice (like bellow)